src/components/home/displayCharacters.js

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable */
```

Comment:
- [ ] Linters are used to flag programming errors, bugs, stylistic errors, and suspicious constructs so kindly remove the `/*eslint-disable*/` command, and try to fix the linters errors so your code stays readable and clean